packages/errors/src/context.ts

```diff
@@ -533,6 +543,43 @@ export type SolanaErrorContext = DefaultUnspecifiedErrorContextToUndefined<
         [SOLANA_ERROR__NONCE_ACCOUNT_NOT_FOUND]: {
             nonceAccountAddress: string;
         };
+        [SOLANA_ERROR__OFFCHAIN_MESSAGE__ADDRESSES_CANNOT_SIGN_OFFCHAIN_MESSAGE]: {
+            expectedAddresses: string[];
+            unexpectedAddresses: string[];
+        };
+        [SOLANA_ERROR__OFFCHAIN_MESSAGE__APPLICATION_DOMAIN_STRING_LENGTH_OUT_OF_RANGE]: {
+            actualLength: number;
+        };
+        [SOLANA_ERROR__OFFCHAIN_MESSAGE__ENVELOPE_SIGNERS_MISMATCH]: {
+            missingRequiredSigners: readonly string[];
+            unexpectedSigners: readonly string[];
```

Comment:
There are the only two `readonly string[]` in the added context. The others are `string[]`. Should we make them all `readonly string[]` for consistency?